modules/openideal_user/src/Form/OpenidealSecondStepRegistrationForm.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\openideal_user\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Url;
+
+/**
+ * OpenidealSecondStepRegistration object.
+ */
+class OpenidealSecondStepRegistrationForm extends ContentEntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'user-additional-details';
```

Comment:
There is a common practice to add form ID with underscores instead of dashes: https://www.drupal.org/docs/8/api/form-api/introduction-to-form-api#s-getformid